wire/msgheaders_test.go

```diff
@@ -308,6 +313,7 @@ func TestHeadersWireErrors(t *testing.T) {
 		0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, // SBits
 		0x01, 0xe3, 0x62, 0x99, // Nonce
 		0x01, // TxnCount (should be 0 for headers message, but 1 to force error)
+		//0xf0, 0x1d, 0xab, 0x1e, // XXX should we test StakeVersion?
```

Comment:
Yes.  And it needs to be in the correct spot, which would be before the TxnCount.